src/components/Edit/ChangeSummary.vue

```diff
@@ -4,33 +4,9 @@
     <div class="summary-header px-4 mb-2 rounded-t">
       <v-row no-gutters>
         <v-col cols="9">
-          <v-icon color="primary">mdi-file-document-edit-outline</v-icon>
+          <v-icon class="header-icon">mdi-file-document-edit-outline</v-icon>
           <label class="summary-title">Summary of Changes to File</label>
         </v-col>
-
-        <!-- Actions -->
-        <v-col cols="3" class="mt-n2">
-          <div class="actions mr-4">
-            <v-btn
-              text color="primary"
-              id="btn-change"
-              :disabled="isBusySaving"
-              @click="onChangeClicked()"
```

Comment:
Can you now delete onChangeClicked() and onDeleteClicked() ?

Reply:
Indeed!

This is what happens when i get cater to ongoing UX feedback, get caught in a spiral of small updates outside the scope of my tickets 😅 

Will remove!

Reply:
And also onDeleteClicked() ?

Reply:
Was a weird one, my editor said it was still in use!
removed (until i add i back in..)